src/Datadog.Trace.ServiceFabric.Remoting/Datadog.Trace.ServiceFabric.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Nullable>enable</Nullable>
+
+    <!-- Microsoft.ServiceFabric.Services.Remoting only supports x64 -->
+    <Platforms>x64</Platforms>
+
+    <!-- NuGet -->
+    <Version>0.1.0-alpha1</Version>
```

Comment:
TODO: update the versioning app to update this project

Reply:
update version manually because I want to use multiple pre-release suffixes (`alpha1`, `alpha2`, etc) while keeping the number in sync with `Datadog.Trace`.